assets/package.json

```diff
@@ -86,8 +92,10 @@
     "date-time-format-timezone": "1.0.22",
     "element-closest": "3.0.2",
     "esri-leaflet": "2.3.3",
+    "exports-loader": "0.7.0",
```

Comment:
I don't think we are using this. Please remove

Reply:
Got rid of exports loader.